src/main/java/org/klodnicki/controller/AccountController.java

```diff
@@ -55,4 +59,27 @@ public void showAccounts() {
             menuController.displayOnMenu(e.getMessage());
         }
     }
+
+    public void removeAccount() {
+        menuController.displayOnMenu(REMOVE_WARNING);
+        showAccounts();
+        String pesel = menuController.displayOnMenuAndAskForInput(REMOVE_ACCOUNT);
+        String responseConfirmation = menuController.displayOnMenuAndAskForInput(ASK_FOR_CONFIRMATION);
+
+        if (responseConfirmation.equalsIgnoreCase("no")) {
```

Comment:
This can be handled better. I think this is actually similar to the case before where you also made a if for yes/no question. What you can do to make this code more readable is to group together no and "not yes" options into one if and then you don't need to have else in 81st line. This change would keep code clean and easy to read.

Reply:
I improved that block of code, check it out now